installers/osie/main.go

```diff
@@ -96,6 +99,13 @@ func kernelParams(ctx context.Context, action, state string, j job.Job, s ipxe.S
 	s.Args("packet_state=${state}")
 	s.Args("osie_vendors_url=" + conf.OsieVendorServicesURL)
 
+	// Add extra kernel args
+	for _, kv := range i.ExtraKernelArgs {
+		if strings.Contains(kv, "=") {
+			s.Args(kv)
+		}
+	}
```

Comment:
this should probably be a simple `if i.ExtraKernelArgs != "" {` instead.

Reply:
updated